src/cascadia/TerminalSettingsEditor/Profiles.cpp

```diff
@@ -0,0 +1,200 @@
+        INITIALIZE_BINDABLE_ENUM_SETTING(ScrollState, ScrollbarState, winrt::Microsoft::Terminal::TerminalControl::ScrollbarState, L"Profile_ScrollbarVisibility", L"Content");
+
+        // manually add Custom FontWeight option. Don't add it to the Map
+        INITIALIZE_BINDABLE_ENUM_SETTING(FontWeight, FontWeight, uint16_t, L"Profile_FontWeight", L"Content");
+        _CustomFontWeight = winrt::make<EnumEntry>(RS_(L"Profile_FontWeightCustom/Content"), winrt::box_value<uint16_t>(0u));
+        _FontWeightList.Append(_CustomFontWeight);
+
+        // manually keep track of all the Background Image Alignment buttons
+        _BIAlignmentButtons.at(0) = BIAlign_TopLeft();
+        _BIAlignmentButtons.at(1) = BIAlign_Top();
+        _BIAlignmentButtons.at(2) = BIAlign_TopRight();
+        _BIAlignmentButtons.at(3) = BIAlign_Left();
+        _BIAlignmentButtons.at(4) = BIAlign_Center();
+        _BIAlignmentButtons.at(5) = BIAlign_Right();
+        _BIAlignmentButtons.at(6) = BIAlign_BottomLeft();
+        _BIAlignmentButtons.at(7) = BIAlign_Bottom();
+        _BIAlignmentButtons.at(8) = BIAlign_BottomRight();
+    }
+
+    void Profiles::OnNavigatedTo(const NavigationEventArgs& e)
+    {
+        _State = e.Parameter().as<Editor::ProfilePageNavigationState>();
+
+        const auto& colorSchemeMap{ _State.Schemes() };
+        for (const auto& pair : colorSchemeMap)
+        {
+            _ColorSchemeList.Append(pair.Value());
+        }
+
+        const auto& biAlignmentVal{ static_cast<int32_t>(_State.Profile().BackgroundImageAlignment()) };
+        for (const auto& biButton : _BIAlignmentButtons)
+        {
+            biButton.IsChecked(biButton.Tag().as<int32_t>() == biAlignmentVal);
+        }
+    }
+
+    ColorScheme Profiles::CurrentColorScheme()
+    {
+        const auto schemeName{ _State.Profile().ColorSchemeName() };
+        if (const auto scheme{ _State.Schemes().TryLookup(schemeName) })
+        {
+            return scheme;
+        }
+        else
+        {
+            // This Profile points to a color scheme that was renamed or deleted.
+            // Fallback to Campbell.
+            return _State.Schemes().TryLookup(L"Campbell");
+        }
+    }
+
+    void Profiles::CurrentColorScheme(const ColorScheme& val)
+    {
+        _State.Profile().ColorSchemeName(val.Name());
+    }
+
+    fire_and_forget Profiles::BackgroundImage_Click(IInspectable const&, RoutedEventArgs const&)
+    {
+        auto lifetime = get_strong();
+
+        FileOpenPicker picker;
+
+        _State.WindowRoot().TryPropagateHostingWindow(picker); // if we don't do this, there's no HWND for it to attach to
+        picker.ViewMode(PickerViewMode::Thumbnail);
+        picker.SuggestedStartLocation(PickerLocationId::PicturesLibrary);
+        picker.FileTypeFilter().ReplaceAll({ L".jpg", L".jpeg", L".png", L".gif" });
```

Comment:
Just updated the list according to [BitmapImage docs](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Media.Imaging.BitmapImage?view=winrt-19041#remarks) since we convert the URI into a BitmapImage down in TermControl.